__tests__/test.js

```diff
@@ -24,4 +35,15 @@ describe("test DB for requests", () => {
         });
       });
   });
+  test("serves up a json representation of all the available endpoints of the api", () => {
+    return request(app)
+      .get("/api")
+      .then(async (res) => {
+        expect(res.status).toBe(200);
+        const filePath = path.join(__dirname, "..", "endpoints.json");
+        const data = await fs.readFile(filePath, "utf-8");
+        const apiEndpoints = JSON.parse(data);
+        expect(res.body).toEqualapiEndpoints;
```

Comment:
Defo mistake on like 46 which is producing a false positive. This needs to be fixed asap and resubmitted.